calcom/cal.com
# fix: missing videoCallUrl in metadata for static link based video apps

## What does this PR do?

VideoCallUrl wasn't saved in booking metadata for static link-based video apps like Wherby. That's why it also didn't appear in workflow emails when using the {MEETING_URL} variable

Fixes #13972 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- set event type location to Wherby 
- book event 
- check metadata in db
